tlscanary/template/js/report_page.js

```diff
@@ -241,59 +229,59 @@ function getVisibleColumns()
   return columns;
 }
 
-function getSortedRows()
-{
+function getSortedRows() {
   var gridData = $('#grid').bootgrid().data('.rs.jquery.bootgrid').rows;
   var currentRows = [];
   var searchStr = $('#grid').bootgrid("getSearchPhrase");
   var currentColumns = getVisibleColumns();
-  for (var i=0;i<gridData.length;i++)
+  for (var i = 0;i < gridData.length;i++)
   {
     var row = gridData[i];
-    for (var j=0;j<currentColumns.length;j++)
+    for (var j = 0;j < currentColumns.length;j++)
     {
       var field = row[currentColumns[j]].toString();
-      if ( field.search(searchStr) != -1 )
+      if (field.search(searchStr) != -1)
       {
-        currentRows.push (row);
+        currentRows.push(row);
         break;
       }
     }
   }
   return currentRows;
 }
 
-function make_table(hosts, columns)
-{
+function makeTable(hosts, columns) {
   // First, add new column for our grid actions
-  columns.push ({
-    name:"Actions",
-    default:true,
-    type:null,
-    width:"20%"
-  })
+  columns.push(
+    {
+      name: "Actions",
+      default: true,
+      type: null,
+      width: "20%"
+    }
+  );
   var html = "<table id=\"grid\" class=\"table table-condensed table-hover table-striped\"><thead><tr>";
-  for (var i=0;i<columns.length;i++)
+  for (var i = 0;i < columns.length;i++)
   {
     html += "<th data-column-id=\"" + columns[i].name + "\" ";
-    if (columns[i].name == "rank")
+    if (columns[i].name === "rank")
     {
       html += "data-order=\"asc\" ";
       html += "data-identifier=\"true\" ";
     }
-    if (columns[i].name == "Actions")
+    if (columns[i].name === "Actions")
     {
       html += "data-visible-in-selection=\"false\" data-formatter=\"commands\" data-searchable=\"false\" ";
     }
     if (!columns[i].default)
     {
       html += "data-visible=\"false\" ";
     }
-    if (columns[i].type == "int")
+    if (columns[i].type === "int")
```

Comment:
I notice a mixture of quoting styles. We should stick to one unless it really makes sense to use the other (`'"foo" "bar"` vs. `"\"foo\" \"bar\""`), and I think in JS – like python – there is actually no difference between hard `'` and soft `"`.